metadata.yaml

```diff
@@ -32,7 +32,7 @@ resources:
   integration-hub-image:
     type: oci-image
     description: OCI image for the Spark Integration Hub Charm
-    upstream-source: ghcr.io/canonical/spark-integration-hub@sha256:498f035278d7973990990ec9541f5736ddf460d93d5dc34fb4344bb9f6c68fd9  # 3-22.04 2025-08-20
+    upstream-source: batalex/hub-k8s:latest  # tmp
```

Comment:
Built using the latest commit on https://github.com/canonical/spark-integration-hub-rock/pull/15